deploy/configs/load_balancer/nodes_data_conf.json

```diff
@@ -36,6 +36,7 @@
     "port": "80"
   },
   "host_port": {
+    "hostname": "localhost",
```

Comment:
conceptually, the placement of hostname under host_port would be inappropriate. Given that we give this field a name **load_balancer** in [conf.json](https://github.com/AutolabJS/AutolabJS/blob/master/deploy/configs/execution_nodes/conf.json#L1) of execution nodes, it might be appropriate to use the same object structure here also.

Reply:
Having a field **load_balancer** seems correct.